source/_components/binary_sensor.insteon_plm.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: "Insteon PLM Binary Sensor"
```

Comment:
Hi @shaftoe, you seem to have some unrelated commits in your pull request. Can you please rebase your one commit?